src/shared/components/results/ResultsView.tsx

```diff
@@ -46,6 +43,7 @@ import { ViewMode } from './ResultsContainer';
 
 import './styles/warning.scss';
 import './styles/results-view.scss';
+import usePagination from '../../hooks/usePagination';
```

Comment:
Can you move this up with the rest of the custom hooks?

Reply:
Damn, I really need to write something to check my imports order 😬

Reply:
I mean, I believe that's what the `I have reviewed my own code` bit is for. For me I do this by having a look at the GitHub side-by-side diff view that you get even during the process of creating a PR and go line by line to see if there's anything I've missed.